notebooks/bedrock/Query command XL.ipynb

```diff
@@ -74,16 +75,15 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 18,
+   "execution_count": 1,
    "metadata": {},
    "outputs": [],
    "source": [
     "import os\n",
     "\n",
     "# Replace with your credentials\n",
     "os.environ['AWS_ACCESS_KEY_ID'] = \"your key id\"\n",
```

Comment:
if I remember, `aws configure` in the [quick start](https://github.com/cohere-ai/cohere-aws#quick-start) is enough?

Reply:
you're right - didn't realize those instructions were there